package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -352,6 +377,12 @@ class H5MDReader(base.ReaderBase):
         }
     }
 
+    @due.dcite(Doi("10.25080/majora-1b6fd038-005"),
```

Comment:
Or I guess my comment didn't get added to my review - was wondering if this needed to be added to test_duecredit in some way?

Reply:
I did test it manually with a script that used H5MD and it works. I asked @orbeckst whether a unit test was necessary and he said no

Reply:
Ok, sounds fair, wouldn't have thought it would be a blocking thing either way anyways. @orbeckst do we maybe want to raise an issue for the future to check if we should eventually add to test_duecredit? (sounds like an easy GSoC starter?)

Reply:
I didnt' think we tested any of the other due citations. BUT: I was wrong! 

@edisj if you could add an entry in [testsuite/MDAnalysisTests/utils/test_duecredit.py](https://github.com/MDAnalysis/mdanalysis/blob/develop/testsuite/MDAnalysisTests/utils/test_duecredit.py) then that would be the dot on the i.

(I don't know if we have really tested all duecredits. THat's something to check & easy enough)

Reply:
duecredit test is in